src/core/reopt.jl

```diff
@@ -358,9 +358,18 @@ function build_reopt!(m::JuMP.AbstractModel, p::REoptInputs)
         end
     end
 	
+	# indicator function to say: if the energy capacity is greater than zero, then the dvBatteryIncluded binary is 1
```

Comment:
I think we should avoid adding indicator constraints that will be applied in every REopt run (or even every run with BESS) since some solvers struggle with these. @zolanaj do you have any suggestions for the best way to reconfigure this constraint to not be an indicator constraint?

Reply:
As a temporary solution before the implementation of the Big-M constraint, I added an if statement which ensures that the indicator constraints will only be applied if the install_cost_constant or replace_cost_constant is not zero.

Reply:
I removed the indicator constraints, and reformulated the equations so they don't need the Big-M constraint either